apps/api/src/app/shared/commands/project.command.ts

```diff
@@ -3,18 +3,20 @@ import { BaseCommand } from './base.command';
 
 export { BaseCommand };
 
-export abstract class EnvironmentWithUserCommand extends BaseCommand {
+export abstract class EnvironmentCommand extends BaseCommand {
```

Comment:
Making the command definitions dryer. The key point here is that the `EnvironmentWithUserCommand` requires environmentId presense and raises an error otherwise.